examples/arduino/ATMEGA328P/ATMEGA328P.ino

```diff
@@ -0,0 +1,158 @@
+/**
+ * This file is part of Ark Cpp Crypto.
+ *
+ * (c) Ark Ecosystem <info@ark.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ **/
+
+/**
+ * ATMEGA328P Cpp-Crypto Usage Example Sketch
+ *
+ * This sketch covers how to use the Cpp-Crypto library.
+ * It allows your ATMEGA328P use Ark Ecosystem cryptographic protocols.
+ */
+
+ /**
+ * NOTE: At the time of this writing, the Cpp-Crypto library requires running the 'ARDUINO_IDE.sh' bash script located in the 'extras' folder.
+ * This converts our library to be compatible with the Arduino IDE.
+ */
+
+/****************************************/
```

Comment:
`<ArduinoSTL.h>` and it has to be included before the `<arkCrypto.h>` include 👍 

```
/****************************************/

/**
 * Include the avr_stl header file to bring in the C++ STL
 */
  #include <ArduinoSTL.h>
/**/

/**
 * This is where you include the 'arkCrypto.h' header.
 * This allows your project to use Ark Cpp-Crypto.
 */
#include <arkCrypto.h>
/**/

/****************************************/
```

Reply:
Since the library name is now avr_stl, I went ahead and made a properly named file <avr_stl.h>.  I will run CI again once v1.2.2 makes its way through the Arduino library system.

ArduinoSTL.h pulls in a lot of stuff that we don't need/want by default.  Gonna try to get a more minimalist solution working.

If that fails, I can revisit this.